app/views/gobierto_participation/processes/_process_teaser.html.erb

```diff
@@ -0,0 +1,9 @@
+<div class="process pure-u-1 pure-u-md-1-2">
+  <h3><%= link_to process.title, gobierto_participation_process_path(process.slug) %></h3>
+  <div class="meta">
+    <svg xmlns="http://www.w3.org/2000/svg" width="40" height="12" viewBox="0 0 40 12">
+      <path fill="none" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M473.55201,778 C472.783025,776.234095 471.026818,775 468.983333,775 C466.231114,775 464,777.238576 464,780 C464,782.761424 466.231114,785 468.983333,785 C470.812002,785 472.410617,784.011729 473.277582,782.538462 L478.689085,782.538462 C479.556049,784.011729 481.154665,785 482.983333,785 C484.812002,785 486.410617,784.011729 487.277582,782.538462 L492.689085,782.538462 C493.556049,784.011729 495.154665,785 496.983333,785 C499.735552,785 501.966667,782.761424 501.966667,780 C501.966667,777.238576 499.735552,775 496.983333,775 C494.939849,775 493.183642,776.234095 492.414657,778 L487.55201,778 C486.783025,776.234095 485.026818,775 482.983333,775 C480.939849,775 479.183642,776.234095 478.414657,778 L473.55201,778 Z" opacity=".7" transform="translate(-463 -774)"/>
+    </svg>
+    <%= t('.current_stage_html', current_stage: process.current_stage) %>
```

Comment:
¿Can you modify the `current_stage` method so when the process is already closed it returns the latest stage?

Reply:
I corrected the current_stage method, had the wrong comparators and already shows the correct information.